components/scream/src/physics/p3/atmosphere_microphysics.hpp

```diff
@@ -246,9 +247,11 @@ class P3Microphysics : public AtmosphereProcess
         diag_eff_radius_qc(icol,ipack) *= 1e6;
         diag_eff_radius_qi(icol,ipack) *= 1e6;
       } // for ipack
+      precip_liq_surf(icol) += precip_liq_surf_local(icol) * PC::RHO_H2O * m_dt;
+      precip_ice_surf(icol) += precip_ice_surf_local(icol) * PC::RHO_H2O * m_dt;
```

Comment:
dumb question, but isn't precip in m/s = precip mass in kg *1/RHO_H2O/dt? I don't understand why you're multiplying here.

Reply:
also, maybe call _local _flux instead?

Reply:
Oh, my 2nd comment answers my first: The RHS is m/s and the LHS is in kg. Nevermind...